packages/manager/apps/vrack-services/src/components/UpdateMessages.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { OsdsMessage, OsdsText } from '@ovhcloud/ods-components/react';
+import {
+  ODS_MESSAGE_TYPE,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+} from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { useTranslation } from 'react-i18next';
+import { useMutationState } from '@tanstack/react-query';
+import { useVrackServicesList } from '@/utils/vs-utils';
+import {
+  ResourceStatus,
+  VrackServicesWithIAM,
+  updateVrackServicesQueryKey,
+} from '@/api';
+import { getSubnetCreationMutationKey } from '@/pages/subnets/constants';
+import { getEndpointCreationMutationKey } from '@/pages/endpoints/constants';
+
+const shouldDisplayMessage = (vs: VrackServicesWithIAM) =>
+  [
+    ResourceStatus.CREATING,
+    ResourceStatus.UPDATING,
+    ResourceStatus.DELETING,
+    ResourceStatus.ERROR,
+  ].includes(vs.resourceStatus);
+
+const OperationMessage: React.FC<{ vs: VrackServicesWithIAM }> = ({ vs }) => {
```

Comment:
You should stay consistent between the file name and the class exported in the file.
Either this file shoud be named OperationMessage.tsx or class name should be UpdateMessages

Reply:
Thank you, it's fixed